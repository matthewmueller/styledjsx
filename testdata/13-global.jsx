import SlackButton from "./slack-button"
import Button from "./button"

export default function Header({ success }) {
  return (
    <header>
      <nav className="buttons">
        {/* <Button href={`mailto:${props.email}`}>Contact</Button> */}
        <Button href="/faq">FAQ</Button>
        <SlackButton success={success} />
      </nav>
      <style jsx>
        {`
          .buttons {
            display: flex;
            align-items: center;
            position: absolute;
            right: 10px;
            top: 20px;
          }

          .buttons > :global(*) {
            margin-right: 15px;
          }

          .buttons > :global(*:last-child) {
            margin-right: 0px;
          }
        `}
      </style>
    </header>
  )
}
