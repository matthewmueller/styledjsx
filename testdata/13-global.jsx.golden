import Style from "styled-jsx";
import SlackButton from "./slack-button"
import Button from "./button"

export default function Header({ success }) {
  return (
    <header class="jsx-zVKDd">
      <nav className="jsx-zVKDd buttons">
        {/* <Button href={`mailto:${props.email}`}>Contact</Button> */}
        <Button href="/faq">FAQ</Button>
        <SlackButton success={success} />
      </nav>
      <Style jsx id="jsx-zVKDd">{`.buttons.jsx-zVKDd{display:flex;align-items:center;position:absolute;right:10px;top:20px}.buttons.jsx-zVKDd>*{margin-right:15px}.buttons.jsx-zVKDd>*:last-child{margin-right:0}`}</Style>
    </header>
  )
}
